terraform/environments/corporate-staff-rostering/ssm-documents/ami-build.yaml

```diff
@@ -16,30 +16,54 @@ parameters:
     description: "Name of the AMI being built"
 
 mainSteps:
-  - name: CreateSnapshot 
-    action: aws:executeAWSApi
-    onFailure: Abort 
+  - name: installAWSToolsForPowershell
+    action: aws:runPowerShellScript
     inputs:
-      Service: EC2
-      API: CreateSnapshot
-      VolumeId: "{{ VolumeId }}"
-      Description: "{{ SnapshotName }}"
-    outputs:
-    - Name: Payload
-      Selector: SnapshotId
-      Type: String
+      runCommand:
+        - |
+          [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+          Import-Module AWSPowerShell - Force
```

Comment:
you need to take the space out of here between - and Force

Reply:
```suggestion
          Import-Module AWSPowerShell -Force
```